bigtop-tests/spec-tests/runtime/src/test/resources/testRuntimeSpecConf.groovy

```diff
@@ -300,7 +300,7 @@ specs {
       name = 'HADOOP_GETCONF'
       type = 'shell'
       arguments {
-        command = '[ `hdfs getconf -confKey dfs.permissions.superusergroup` == "hadoop" ]'
+        command = '[ `hdfs getconf -confKey dfs.permissions.superusergroup >/dev/null 2>/dev/null; echo $?` == "0" ]'
```

Comment:
`>/dev/null 2>/dev/null;` could be reduced to `2>&1 > /dev/null;`
I don't mind either way. Looks good otherwise.